nova_vm/src/ecmascript/types/language/value.rs

```diff
@@ -181,6 +174,11 @@ impl Value {
         )
     }
 
+    pub fn is_function(self) -> bool {
+        matches!(self, |Value::BoundFunction(_)| Value::BuiltinFunction(_)
```

Comment:
I have no idea how this leading pipe happened, but I don't think it's correct...
```suggestion
        matches!(self, Value::BoundFunction(_) | Value::BuiltinFunction(_)
```